README.md

```diff
@@ -18,17 +18,16 @@ Refer to the Cortex Certifai documentation: https://cognitivescale.github.io/cor
-| [ScanFromDefinition.ipynb](https://github.com/CognitiveScale/cortex-certifai-examples/tree/master/notebooks/scan_from_definition) - Running the scanner within a notebook from a definition file  | Illustrates the use of Certifai to generate reports based upon an existing scan definition, with multiple models and multiple analysis types which can be run as a single evaluation.   | Binary classification  | Logistic Regression  |  Fairness |   |
+| [ScanFromDefinition.ipynb](https://github.com/CognitiveScale/cortex-certifai-examples/tree/master/notebooks/scan_from_definition) - Running the scanner within a notebook from a definition file  | Illustrates the use of Certifai to generate reports based upon an existing scan definition, with multiple models and multiple analysis types which can be run as a single evaluation.   | Binary classification  | Logistic Regression  |  Fairness | Using the scanner from a notebook |
 | [SHAP.ipynb](https://github.com/CognitiveScale/cortex-certifai-examples/tree/master/notebooks/shap) - SHAP explanations | Illustrates the use of an alternative explanation type (Kernel SHAP) to provide both an explainability measure for a model and explanations for individual predictions. SHAP may be used as an alternative to Certifai counterfactual explanation in circumstances where feature weights rather than counterfactual exemplars are preferred. | Binary Classification | SVM <br /> Logistic Regression | Explanations  | SHAP Explanations |
-| [xgboostDmatrixExample.ipynb](https://github.com/CognitiveScale/cortex-certifai-examples/tree/master/notebooks/xgboost-model) - Running Cortex Certifai fairness evaluation on xgboost model to predict adult income | Illustrates how to accommodate Xgboost models trained with DMatrix data structure in Certifai Notebooks. | Binary Classification | Xboost (XBG) | Fairness | Xboost models <br /> DMatrix data structure |
+| [xgboostDmatrixExample.ipynb](https://github.com/CognitiveScale/cortex-certifai-examples/tree/master/notebooks/xgboost-model) - Running Cortex Certifai fairness evaluation on xgboost model to predict adult income | Illustrates how to accommodate Xgboost models trained with DMatrix data structure in Certifai Notebooks. | Binary Classification | Xboost (XBG) | Fairness | Handling models with suggested data encodings |
```

Comment:
I'm a little leery of the word 'suggested' here.  Suggested by whom?  In general the issue here is that the choice of model dictates the encoding (XGBoost *requires* the use of DMatrix for example).

Reply:
Changing "suggested" to "implicit"